youtube_dl/extractor/common.py

```diff
@@ -43,6 +43,7 @@
     GeoRestrictedError,
     GeoUtils,
     int_or_none,
+    orderedSet,
```

Comment:
Imported items should be sorted alphanumerically. Note that in this project symbols are case-insensitive during sorting.

(I know there are many places not sorted well; it would be great if an automation tool or someone can fix them...)